notebook/static/base/js/utils.js

```diff
@@ -500,41 +500,13 @@ define([
     };
     
     var to_absolute_cursor_pos = function (cm, cursor) {
-        /**
-         * get the absolute cursor position from CodeMirror's col, ch
-         */
-        if (!cursor) {
-            cursor = cm.getCursor();
-        }
-        var cursor_pos = cursor.ch;
-        for (var i = 0; i < cursor.line; i++) {
-            cursor_pos += cm.getLine(i).length + 1;
-        }
-        return cursor_pos;
+        console.warn('utils.to_absolute_cursor_pos(cm, pos) is deprecated us cm.indexFromPos(cursor)');
```

Comment:
... ` is deprecated us`, you want to put `use`? `by`? I don't get the phrase written as is...

below it is the same... thanks!